diff --git a/node_modules/sql-parser/browser/sql-parser.js b/node_modules/sql-parser/browser/sql-parser.js
index a3cf815..d8e581f 100644
--- a/node_modules/sql-parser/browser/sql-parser.js
+++ b/node_modules/sql-parser/browser/sql-parser.js
@@ -583,15 +583,14 @@ parse: function parse(input) {
         vstack.length = vstack.length - n;
         lstack.length = lstack.length - n;
     }
-    _token_stack:
-        function lex() {
-            var token;
-            token = lexer.lex() || EOF;
-            if (typeof token !== 'number') {
-                token = self.symbols_[token] || token;
-            }
-            return token;
+    function lex() {
+        var token;
+        token = lexer.lex() || EOF;
+        if (typeof token !== 'number') {
+            token = self.symbols_[token] || token;
         }
+        return token;
+    }
     var symbol, preErrorSymbol, state, action, a, r, yyval = {}, p, len, newState, expected;
     while (true) {
         state = stack[stack.length - 1];
